.nsm.mk

```diff
@@ -22,6 +22,7 @@ DOCKER_SIMPLE_DATAPLANE=networkservicemesh/simple-dataplane
 DOCKER_NSM_INIT=networkservicemesh/nsm-init
 DOCKER_NSE=networkservicemesh/nse
 DOCKER_RELEASE=networkservicemesh/release
+DOCKER_SIDECAR_INJECTOR=networkservicemesh/sidecar-injector
```

Comment:
Note that I've created the matching Docker Hub repository here [1].

[1] https://hub.docker.com/r/networkservicemesh/sidecar-injector/

Reply:
Just to be sure, I don't need to do anything for this right? Travis will take care of pushing the image.

Reply:
Correct, travis will do this automatically once your Makefile changes merge.